p2p/peer_tracker.go

```diff
@@ -10,24 +10,26 @@ import (
 	"github.com/libp2p/go-libp2p/core/network"
 	"github.com/libp2p/go-libp2p/core/peer"
 	"github.com/libp2p/go-libp2p/p2p/net/conngater"
+
+	"github.com/celestiaorg/go-header/p2p/peerstore"
 )
 
 const (
 	// defaultScore specifies the score for newly connected peers.
 	defaultScore float32 = 1
-	// maxTrackerSize specifies the max amount of peers that can be added to the peerTracker.
+	// maxTrackerSize specifies the max amount of peers that can be added to the PeerTracker.
 	maxPeerTrackerSize = 100
 )
 
 var (
 	// maxAwaitingTime specifies the duration that gives to the disconnected peer to be back online,
 	// otherwise it will be removed on the next GC cycle.
 	maxAwaitingTime = time.Hour
-	// gcCycle defines the duration after which the peerTracker starts removing peers.
-	gcCycle = time.Minute * 30
+	// gcCycle defines the duration after which the PeerTracker starts removing peers.
+	gcCycleDefault = time.Minute * 1
 )
 
-type peerTracker struct {
+type PeerTracker struct {
```

Comment:
The purpose of this change was to prevent having to pass in params that tracker-specific to the exchange (that doesn't necessarily care about them) such as connGater and peerstore.